examples/mptest/mptest.c

```diff
@@ -55,11 +56,11 @@ int main(void)
                         {
                             if (err == -3)
                             {
-                                printf("Mempak is not formatted!");
+                                printf("Pak is not formatted!");
```

Comment:
What's the reason for using `Pak` here rather than `CPak`?

Reply:
IMO it is already assumed by the menu option, although happy to change... let me know.